content/api/commands/mount.md

```diff
@@ -0,0 +1,607 @@
+include the `cypress.d.ts` file in all your `tsconfig.json` files like so:
+
+```json
+"include": ["./src", "cypress.d.ts"]
+```
+
+## Additional Mount Commands
+
+You're not limited to a single `cy.mount()` command. If needed, you can create
+any number of custom mount commands, as long as they have unique names.
+
+Below are some examples for common uses cases and libraries.
+
+## React Examples
+
+If your React component relies on provider to work properly, you will need to
+wrap your component in that provider in your component tests. This is a good use
+case to create a custom mount command that wraps your components for you.
+
+Below are a few examples that demonstrate how. These examples can be adjusted
+for most other providers that you will need to support.
+
+### React Router
+
+If you have a component that consumes a hook or component from
+[React Router](https://reactrouter.com/), you will need to make sure the
+component has access to a React Router provider. Below is a sample mount command
+that uses `MemoryRouter` to wrap the component. Setup props for `MemoryRouter`
+can be passed in the options param as well:
+
+```jsx
+import { mount } from '@cypress/react'
+import { MemoryRouter } from 'react-router-dom'
+
+Cypress.Commands.add('mountWithRouter', (component, options = {}) => {
+  const { routerProps = { initialEntries: ['/'] }, ...mountOptions } = options
+
+  const wrapped = <MemoryRouter {...routerProps}>{component}</MemoryRouter>
+
+  return mount(wrapped, mountOptions)
+})
+```
+
+Typings:
+
+```ts
+import { MountOptions, MountReturn } from '@cypress/react'
+import { MemoryRouterProps } from 'react-router-dom'
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+       * Mounts a React node
+       * @param jsx React Node to mount
+       * @param options Additional options to pass into mount
+       */
+      mountWithRouter(
+        jsx: React.ReactNode,
+        options?: MountOptions & { routerProps?: MemoryRouterProps }
+      ): Cypress.Chainable<MountReturn>
+    }
+  }
+}
+```
+
+In this setup, you can pass in custom props for the `MemoryRouter` in the
+`options` param. Below is an example test that ensures an active link has the
+correct class applied to it by initializing the router with `initialEntries`
+pointed to a particular route:
+
+```jsx
+it('login link should be active when url is "/login"', () => {
+  cy.mountWithRouter(<Navigation />, {
+    routerProps: {
+      initialEntries: ['/login'],
+    },
+  })
+  cy.get('a').contains('Login').should('have.class', 'active')
+})
+```
+
+### Redux
+
+To use a component that consumes state or actions from a
+[Redux](https://react-redux.js.org/) store, you can create a `mountWithRedux`
+command that will wrap your component in a Redux Provider:
+
+```jsx
+import { mount } from '@cypress/react'
+import { Provider } from 'react-redux'
+
+Cypress.Commands.add('mountWithRedux', (component, store, options = {}) => {
```

Comment:
Would someone ever want there to be a default `store`? If so, I'd recommend moving it to be a property inside `options` with a default value of `getStore()`. Otherwise, it might be clearer to rename `options` here to `mountOptions`.